README.md

```diff
@@ -25,16 +25,31 @@ The MyDRTV platform is designed to provide:
 ## Architectural Approach
 
 ### Architectural Styles Comparison
-We considered multiple architectural styles for the MyDRTV project. Below is a summary of the criteria used in our comparison:
+We considered multiple architectural styles for the MyDRTV project. Below is a summary of the criteria used in our comparison: 
 
 ## Technology Stack
 
-- **Frontend:** [Frontend technology, e.g., React, Angular]
-- **Backend:** [Backend technology, e.g., Node.js, Django]
-- **Database:** [Database, e.g., PostgreSQL, MongoDB]
-- **Search Engine:** [Search technology, e.g., Elasticsearch]
-- **Recommendation Engine:** [Algorithm or tech for recommendations]
-- **Infrastructure:** Cloud-hosted on [e.g., AWS, Azure] to ensure scalability and high availability.
+- **Frontend:** Typescript React. This choose is made over Javascript becuase we would like strongly typed language and gives better tooling then Javascript. Then React was choosen because it makes coding dynamic applications easier to write and more performant aswell as when we're already familiar with react beforehand.
+
+- **Backend:** We have choosen C# because its obejct-oriented and uses dotnet which gives access to Entity Framework (EF). EF is a object-relation mapper which makes it easier to make CRUD operations in a relational database. Other programming langauges also have object-relation mapper like JPA for Java but we like how easy and intuative EF is to use.
```

Comment:
```suggestion
- **Backend:** We chose C# because it's object-oriented and utilizes .NET which provides access to Entity Framework (EF). EF is an object-relational mapper (ORM) that simplifies performing CRUD operations on a relational database. While other programming langauges also offer the use of ORMs, such as JPA for Java, We prefer EF for its ease of use and intuitive design.
```